povm_toolbox/post_processor/dual_from_empirical_frequencies.py

```diff
@@ -18,8 +18,9 @@
 from qiskit.quantum_info import DensityMatrix, SparsePauliOp, Statevector
 
 from povm_toolbox.post_processor.povm_post_processor import POVMPostProcessor
-from povm_toolbox.quantum_info import BaseDUAL, ProductPOVM
-from povm_toolbox.quantum_info.product_dual import ProductDUAL
+from povm_toolbox.quantum_info import ProductPOVM
+from povm_toolbox.quantum_info.base import BaseDual
+from povm_toolbox.quantum_info.product_dual import ProductDual
```

Comment:
Should we now keep importing `ProductDual` from `povm_toolbox.quantum_info.product_dual` or change to import directly from `povm_toolbox.quantum_info` (as we do for `ProductPOVM`) ? 
Few other examples in other files (also with `MultiQubitDual` for instance)

Reply:
Oh yeh we can align the imports now. Will do this in a sec